components/mentorPage/FAQ.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+function FAQ({ ques, ans }) {
+  const [open, setOpen] = useState(false);
+  const toggleAns = () => setOpen(!open);
+
+  return (
+    <div className="gray-bg mb-20 rounded container py-5 text-justify">
+      <div className="row justify-content-between align-items-center">
+        <p className="h2 col-10 col-sm-11">{ques}</p>
+        <div className="col-1 cursor-pointer" onClick={toggleAns}>
+          {open ? (
+            <i class="fas fa-chevron-up"  />
+          ) : (
```

Comment:
Make it `setOpen` not only click on pointer but to on the whole container....clicking anywhere open